SPECS/liburing/liburing.spec

```diff
@@ -1,14 +1,13 @@
 Summary:        Linux-native io_uring I/O access library
 Name:           liburing
-Version:        2.0
```

Comment:
It looks like [this upgrade breaks ABI](https://github.com/axboe/liburing/issues/724). Reverse dependencies will need to have their release number bumped up so that we can rebuild and release them against the new `liburing` version.